### PR TITLE
{vis}[GCCcore/11.2.0] Wayland v1.21.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login1.barbora.it4i.cz - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 3.6.8
See https://gist.github.com/3a7ffc90081e8ab535dff36c808cb385 for a full test report.

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login1.karolina.it4i.cz - Linux CentOS Linux 7 (Core), x86_64, AMD EPYC 7452 32-Core Processor, Python 3.10.4
See https://gist.github.com/e1a6cf015a15b75411e930812671c243 for a full test report.

File: easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Author: Jasper Grimm (UoY)
+easyblock = 'Bundle'
+
+name = 'Wayland'
+version = '1.20.0'
```
Comment (boegel) [inline easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.2.0.eb:5]: @sassy-crick Any reason you're sticking to 1.20.0 when 1.21.0 is available already too?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.2.0.eb:5]: see https://lists.freedesktop.org/archives/wayland-devel/2022-June/042268.html

Comment (sassy-crick) [reply easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.2.0.eb]: The idea was to have a bit of 'headroom' as I was not using the latest toolchain version. Thus, if somebody needs it for the latest toolchain, we don't have the same version with different toolchains. That said, following our discussions I have upgraded that. Thanks for the help.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
eb-u2204 - Linux Ubuntu 22.04, x86_64, AMD Ryzen 9 3900X 12-Core Processor, Python 3.10.4
See https://gist.github.com/cf12553f3e79a7f815f425e958f808f1 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3144.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ba47564b21329b7662df1c7b97113823 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**